scripts/new_avl.py

```diff
@@ -0,0 +1,569 @@
+##adapted from github of Pavel Grafov: https://github.com/pgrafov
```

Comment:
I don't think I have permission to push to the `Mkebede:master` branch.  @Mkebede or @AmandaKedaigle  can you please add the third party license information?  The Garnet integration test is done, and once we merge that pull request (#25) we'll be ready to merge this one.  We should merge #25 before making new commits here so that the Garnet integration test runs.

I would suggest the following but am open to other ideas:
* Create a `LICENSE-3RD-PARTY` text file in the main Omics Integrator directory.  This would contain the python-avl-tree copyright and MIT license notices from https://github.com/pgrafov/python-avl-tree.
* Add `Third-party code` section at the bottom of the main Omics Integrator `README.md`.  This would refer to `LICENSE-3RD-PARTY` for license details and then have a list of code we use.  For now the only entry would be "python-avl-tree by Pavel Grafov" with the link to https://github.com/pgrafov/python-avl-tree
* At the top of `new_avl.py`  again copy the python-avl-tree copyright and MIT license notices from https://github.com/pgrafov/python-avl-tree as comments.